libs/designsystem/src/lib/components/checkbox/checkbox.component.html

```diff
@@ -4,7 +4,7 @@
     [checked]="checked"
     [disabled]="disabled"
     (ionChange)="onChecked($event.detail.checked)"
-    [attr.aria-labelledby]="labelText ? _labelId : ''"
+    [attr.aria-labelledby]="text ? _labelId : ''"
   ></ion-checkbox>
-  {{ labelText }}
+  {{ text }}
```

Comment:
Consider wrapping the text in a `span` and apply the id/_labelId here instead...